sagemaker-featurestore/feature_store_introduction.ipynb

```diff
@@ -24,21 +24,21 @@
     "This notebook uses both `boto3` and Python SDK libraries, and the `Python 3 (Data Science)` kernel. This notebook works with Studio, Jupyter, and JupyterLab. \n",
     "\n",
     "#### Library dependencies:\n",
-    "* sagemaker>=2.0.0\n",
-    "* numpy\n",
+    "* sagemaker>=2.100.0\n",
+    "* NumPy\n",
```

Comment:
I think the dependency names are based off of the python package (i.e. ones listed on PyPI). It should be kept as numpy.

Reply:
Changed this because numpy will fail grammar check. Better to use code format here. Update in the next PR.